kubernetes/cray-sysmgmt-health/templates/node-exporter/rbac.yaml

```diff
@@ -22,8 +22,6 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 {{- /*
-Copyright 2021 Hewlett Packard Enterprise Development LP
-
 This is used by the node-exporter post-install Hook. The Hook needs to do a
 patch operation on the cray-sysmgmt-health-prometheus-node-exporter DeamonSet.
 */ -}}
```

Comment:
We want to change this comment block as well to avoid {{-

Reply:
This renders as
```
# Source: cray-sysmgmt-health/templates/prometheus/patch-node-exporter/rbac.yaml
kind: ServiceAccount
apiVersion: v1
metadata:
  name: patch-nodeexporter-alerts
  namespace: default
  ```
  So it is not required.